apps/list.c

```diff
@@ -115,8 +112,12 @@ static void list_ciphers(void)
         BIO_printf(bio_err, "ERROR: Memory allocation\n");
         return;
     }
-    BIO_printf(bio_out, "Legacy:\n");
-    EVP_CIPHER_do_all_sorted(legacy_cipher_fn, bio_out);
+#ifndef OPENSSL_NO_DEPRECATED_3_0
+    if (include_legacy) {
+        BIO_printf(bio_out, "Legacy:\n");
```

Comment:
Maybe it should say "Legacy implementations", to be super clear

Reply:
(not just in this spot)

Reply:
I'd prefer to not use the term "legacy" at all here.  It's confusing.
How about "compatibility implementations" instead?

Reply:
The "Legacy" label is what was there BTW :grinning:

Reply:
I'm thinking "Compatibility" might be better.  It sits more nicely with "Provided" which also doesn't have the "implementations".  That or add "implementations" to both (which doesn't seem necessary).

Reply:
And done.  I added a `-backwards-compatibility` option to enable the display of legacy implementations in listings.